packages/shared/src/features/onboarding/shared/CookieConsent.tsx

```diff
@@ -0,0 +1,93 @@
+import type { ReactElement } from 'react';
+import React from 'react';
+import classNames from 'classnames';
+import {
+  Typography,
+  TypographyTag,
+} from '../../../components/typography/Typography';
+import { cookiePolicy, termsOfService } from '../../../lib/constants';
+import { useAuthContext } from '../../../contexts/AuthContext';
+import { otherGdprConsents } from '../../../hooks/useCookieBanner';
+import { LazyModal } from '../../../components/modals/common/types';
+import type { AcceptCookiesCallback } from '../../../hooks/useCookieConsent';
+import { useLazyModal } from '../../../hooks/useLazyModal';
+import { ClickableText } from '../../../components/buttons/ClickableText';
+
+export interface OnboardingCookieConsentProps {
+  onAccepted: AcceptCookiesCallback;
+  onModalClose: () => void;
+  onHideBanner: () => void;
+  className?: string;
+}
+
+export function CookieConsent({
+  onAccepted,
+  onHideBanner,
+  onModalClose,
+  className,
+}: OnboardingCookieConsentProps): ReactElement {
+  const { openModal } = useLazyModal();
+  const { isGdprCovered } = useAuthContext();
+  const onAcceptAll = () => onAccepted(otherGdprConsents);
+
+  const onOpenModal = () => {
+    onHideBanner();
+    openModal({
+      type: LazyModal.CookieConsent,
+      props: { onAcceptCookies: onAccepted, onAfterClose: onModalClose },
+    });
+  };
+
+  return (
+    <div
+      className={classNames(
+        'flex w-full flex-col gap-2 bg-background-default px-4 py-3 text-text-secondary',
+        className,
+      )}
+    >
+      <span
+        className="text-text-tertiary typo-caption2"
+        data-testid="cookie_content"
+      >
```

Comment:
This could have been Typography I guess?
Now you also have duplicate color types in the wrapper above

Reply:
i'm not used to typography 🙈 
we'll take care of that

Reply:
the wrapper above is to set default color for whatever text is inside